### PR TITLE
PBM-1414 Documented changes in NumInsertionWorkers

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   docs/reference/pbm-commands.md
	modified:   docs/reference/restore-options.md
	modified:   docs/usage/restore.md

### CODE REVIEW

File: docs/usage/restore.md [hunk 1]
```diff
@@ -115,19 +115,29 @@ To restore a backup, use the [`pbm restore`](../reference/pbm-commands.md#pbm-re
...
     ```
 
     The default values were adjusted to fit the setups with the memory allocation of 1GB and less for the agent.
 
-    !!! note 
 
-        The lower the values, the less memory is allocated for the restore. However, the performance decreases too.
+    !!! important 
+
+        In version 2.8.0, the default number for the `numInsertionWorkers` parameter has changed. Verify your applications and adjust the configuration if needed.
```
Comment (nastena1606) [inline docs/usage/restore.md:131]:
```suggestion
        In version 2.8.0, the default number for the `numInsertionWorkers` parameter has changed from 10 to 1. Verify your applications and adjust the configuration if needed.
```

File: docs/usage/restore.md [hunk 2]
```diff
@@ -115,19 +115,29 @@ To restore a backup, use the [`pbm restore`](../reference/pbm-commands.md#pbm-re
...
+
+    Starting with version 2.8.0, you can override the number of insertion workers for a specific restore operation. 
+
+    ```{.bash data-prompt="$"}
+    $ pbm restore <backup_name> --numInsertionWorkers 5
```
Comment (nastena1606) [inline docs/usage/restore.md:136]:
```suggestion
    $ pbm restore <backup_name> --num-insertion-workers-per-collection=5
```

**Review by radoslawszulgo — APPROVED** [review]

File: docs/reference/restore-options.md [hunk 3]
```diff
@@ -24,9 +24,9 @@ The number of documents to buffer.
 ### restore.numInsertionWorkers
 
 *Type*: int <br>
-*Default*: 10
+*Default*: 1
```
Comment (radoslawszulgo) [inline docs/reference/restore-options.md:27]: Please revert to 10 now. 
```suggestion
*Default*: 10
```

**Review by radoslawszulgo — CHANGES_REQUESTED** [review]
@nastena1606 please change the default back to 10 for now. Also adjust the "important" section -> leave just we added an option to specify the number of insertion workers per collection via command line

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**